### PR TITLE
fixed the attachment bug in 8.8 server

### PR DESCRIPTION
**Pull request by 4manasa**

### CODE REVIEW

File: src/app/_components/attachment/attachment.component.ts [hunk 1]
```diff
@@ -264,13 +264,14 @@ export class AttachmentComponent implements OnInit {
...
-          this.pConn$.attachmentsInfo = {
+          const reqObj = {
             type: "File",
             attachmentFieldName: this.att_valueRef,
             category: this.att_categoryName,
-            ID: fileRes.ID    // 8.7 - ID is at the top-level now.
+            ID: fileRes.ID
           };
-  
+          this.pConn$.attachmentsInfo = this.PCore$.getPCoreVersion()?.includes('8.8') ? [reqObj] : reqObj;
```
Comment (tumms2021389) [inline src/app/_components/attachment/attachment.component.ts:273]: Does this logic work for the next versions of infinity? I think we should modify this logic to handle two cases

versions below 8.7
versions above 8.7

**Review by tumms2021389 — CHANGES_REQUESTED** [review]

File: src/app/_components/attachment/attachment.component.ts (hunk 1 above)
Comment (4manasa) [reply src/app/_components/attachment/attachment.component.ts]: updated the PR by modifying the condition to support 8.7 and above versions, for versions below 8.7 we have different branch so no need to handle

**Review by JEAxero — APPROVED** [review]
Looks good. We should add in an attachment test at some point.

**Comment by JEAxero** [comment]
@4manasa Note that you need to rebase against main

**Comment by tumms2021389** [comment]
please follow the steps I mentioned here https://github.com/pegasystems/angular-sdk/pull/120#issuecomment-1253615713 for rebase with main branch

**Comment by 4manasa** [comment]
> Looks good. We should add in an attachment test at some point.

sure john

**Review by tumms2021389 — APPROVED** [review]

**Pull request merged**